tests/integration/__snapshots__/formatExports.test.ts.snap

```diff
@@ -4468,15 +4507,16 @@ PackageDownloadLocation: NOASSERTION
 PrimaryPackagePurpose: CONTAINER
 FilesAnalyzed: false
 PackageChecksum: SHA256: shas256:redacted
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: NOASSERTION
 ExternalRef: PACKAGE-MANAGER purl pkg:oci/localhost:5000/match-coverage/debian@sha256:redacted?arch=amd64&tag=latest
 
 ##### Package: apt
 
 PackageName: apt
 
 PackageVersion: 1.8.2
-PackageSupplier: Person: APT Development Team <deity@lists.debian.org>
-PackageOriginator: Person: APT Development Team <deity@lists.debian.org>
+PackageSupplier: NOASSERTION
```

Comment:
This is a bit surprising. Why did we stop asserting the package supplier?

Reply:
I suspect it's due to this PR: https://github.com/anchore/syft/pull/2822, but it's not entirely clear offhand why